docs/ADMIN.md

```diff
@@ -65,6 +65,7 @@ and oauth service to have the same value for them):
 | --vault-secretid-filepath                             | VAULTAPPROLESECRETIDFILEPATH   | /etc/spi/secret_id    | Used with Vault approle authentication. Filepath with secret_id.                                                                                                                                                                   |
 | --vault-k8s-sa-token-filepath                         | VAULTKUBERNETESSATOKENFILEPATH |                       | Used with Vault kubernetes authentication. Filepath to kubernetes ServiceAccount token. When empty, Vault configuration uses default k8s path. No need to set when running in k8s deployment, useful mostly for local development. |
 | --vault-k8s-role                                      | VAULTKUBERNETESROLE            |                       | Used with Vault kubernetes authentication. Vault authentication role set for k8s ServiceAccount.                                                                                                                                   |
+| --vault-data-path-prefix                              | VAULTDATAPATHPREFIX            | spi                   | Path prefix in Vault token storage under which all SPI data will be stored. No leading or trailing '/' should be used, it will be trimmed.                                                                                         |
```

Comment:
maybe it is worth to mention (maybe not right here but somewhere in the doc) that  expected path would be like /{prefix}/data/ ?

Reply:
no, it's just prefix. the `data` part is handled in code https://github.com/redhat-appstudio/service-provider-integration-operator/pull/411/files#diff-9690267b7b50c62307b865c258b7a407dca24db90457879bc6c43a266a9546cfR39